crates/static-analysis-kernel/src/analysis/ddsa_lib/test_utils.rs

```diff
@@ -171,26 +171,54 @@ fn cfg_test_deno_ext() -> deno_core::Extension {
     let mut production_extension = ddsa_lib::init_ops_and_esm();
     let prod_entrypoint = production_extension.get_esm_entry_point().unwrap();
     let prod_ops = production_extension.init_ops().unwrap();
+    #[allow(unused_mut)]
+    let mut ops = prod_ops;
 
     // Clone all ES modules, minus the entrypoint.
     let mut esm_sources = production_extension.get_esm_sources().clone();
     esm_sources.retain(|efs| efs.specifier != prod_entrypoint);
 
+    // Add additional cfg(test) ES modules
+    #[cfg(test)]
+    {
+        use crate::analysis::ddsa_lib::extension::ddsa_lib_cfg_test;
+        let mut cfg_test_extension = ddsa_lib_cfg_test::init_ops_and_esm();
+        let cfg_test_esm_sources = cfg_test_extension.get_esm_sources().clone();
+        let cfg_test_ops = cfg_test_extension.init_ops().unwrap();
+        esm_sources.extend(cfg_test_esm_sources);
+        ops.extend(cfg_test_ops);
+    }
+
     // Create an entrypoint that adds all exports to `globalThis`.
-    let mut entrypoint_code = String::new();
+    let mut entrypoint_code = "'use strict';\n".to_string();
     for (idx, efs) in esm_sources.iter().enumerate() {
-        // Create a unique (arbitrary) variable name for the import.
-        let var_name = "a".repeat(idx + 1);
-        entrypoint_code += &format!(
-            r#"
+        if efs.specifier == "ext:ddsa_lib_cfg_test/helpers" {
+            // Only import under the "TEST" namespace.
+            entrypoint_code += &format!(
+                r#"
+import * as TEST from "{}";
+globalThis.TEST = TEST;
```

Comment:
Why putting this under `TEST`? Why not adding this either with

 - the global scope (`assert(something)`)?
 - the `ddsa` scope `ddsa.assert()`)?